.travis.yml

```diff
@@ -19,24 +19,24 @@ matrix:
   include:
     - os: linux
       compiler: gcc
-      env: REQUIRED_ONLY=--required-only BUILD_TYPE=Debug CODECOV=ON COMPILER=GCC CATKIN_CONFIG_OPTIONS="-DCMAKE_BUILD_TYPE=Debug -DTREAT_WARNINGS_AS_ERRORS=ON" DOWNLOAD_TAGFILES=OFF
+      env: BUILD_NAME=TRUSTY_DEBUG REQUIRED_ONLY=--required-only BUILD_TYPE=Debug COMPILER=GCC CATKIN_CONFIG_OPTIONS="-DCMAKE_BUILD_TYPE=Debug -DTREAT_WARNINGS_AS_ERRORS=ON" DOWNLOAD_TAGFILES=OFF
```

Comment:
Rather than setting `TREAT_WARNINGS_AS_ERRORS = ON` in all these matrix entries, can we just make that the default in `CMakeLists.txt`? I think we also don't need a `DOWNLOAD_TAGFILES` environment variable; we can just set that depending on the `BUILD_NAME` like you did above with `CODECOV`.

Reply:
Maybe we can just set `DOWNLOAD_TAGFILES = ON` in `.ci/build_docs.sh` (not as an environment variable), and then only call that script if `BUILD_NAME = TRUSTY_RELEASE`.

Reply:
> Rather than setting TREAT_WARNINGS_AS_ERRORS = ON in all these matrix entries, can we just make that the default in CMakeLists.txt?

Note that `TREAT_WARNINGS_AS_ERRORS = ON` is part of `CATKIN_CONFIG_OPTIONS` that is used in (`internal-build.sh`](https://github.com/personalrobotics/pr-cleanroom/blob/740e4de50e047f612fb83b763113b5719d71eda1/internal-build.sh#L27).

Reply:
> Maybe we can just set DOWNLOAD_TAGFILES = ON in .ci/build_docs.sh (not as an environment variable), and then only call that script if BUILD_NAME = TRUSTY_FULL_RELEASE.

Hm, how about executing `build_docs.sh` only for `TRUSTY_FULL_RELEASE` and always passing ON for `DOWNLOAD_TAGFILES` in `build_docs.sh`?

Reply:
Great, let's delete `DOWNLOAD_TAGFILES` everywhere in `.travis.yml` then.